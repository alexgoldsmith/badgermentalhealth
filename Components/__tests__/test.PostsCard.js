// https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest
// components/__tests__/Hello.tsx
import React from 'react';
import renderer from 'react-test-renderer';
import PostCard from '../PostsCard'

const mockPost = {
    _id: 3484,
    Title: "Test Title",
    Author: "Test Author",
    Date: new Date(2021, 23, 7),
    Likes: 5,
    NumReplis: 0,
}

it('renders correctly with defaults', () => {
  const component = renderer
    .create(<PostCard post={mockPost}/>)
    .toJSON();
  expect(component).toMatchSnapshot();
});
