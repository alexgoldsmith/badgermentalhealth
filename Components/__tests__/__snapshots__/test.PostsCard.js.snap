// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with defaults 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  nativeID="animatedComponent"
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 10,
      "display": "flex",
      "justifyContent": "center",
      "margin": 20,
      "opacity": 1,
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 24,
      }
    }
  >
    Test Title
  </Text>
  <Text>
    Test Author
  </Text>
  <Text>
    5
     
  </Text>
  <Text />
</View>
`;
